### ISSUE TITLE
Ender chest seems uncraftable

### ISSUE THREAD

**Issue by ScottRadish** [AllTheMods/ATM-Expert#63]
There's a recipe for it in JEI, but putting the items in a crafting grid doesn't craft anything. I suspect it's due to the NBT data on Mekanism's personal chest.

**Comment by MigukNamja** [comment]
That's Mekanism's personal chest. It *should* craft.

Which version of the pack are you running?

**Comment by ScottRadish** [comment]
I just download the newest version to double check. I tested in creative. Here's what's happening. I can craft the ender chest if I pull the Mekanism personal chest out of the creative menu. If I craft the personal chest in survival, I cannot use that chest to craft the ender chest. 


Also, unrelated, the newest version of the pack (1.1.0) seems to have completely borked up the quests from Rockhounding. There are new machines and a multiblock necessary for processing the minerals. Kinda irksome since I had already set it up.
